Kinvey/Kinvey/Realtime.swift

```diff
@@ -124,6 +124,38 @@ public class LiveStream<Type: BaseMappable> {
         return request
     }
     
+    /// Grant access to a user for the `LiveStream`
```

Comment:
As a user of the SDK, how do I retrieve the `acl`? Isn't this class internal?
Also as a side note, the file is still called `Realtime.swift`. Would be nice for it to reflect the new name `LiveService`.

Reply:
No, `LiveStream` is not internal. The name of the file is `Realtime.swift` but there's a public class in it called `LiveStream` and this class is public. I think i got this `LiveStream` from the original Xamarin implementation

Reply:
Yeah, I was referring to the file name being `Realtime.swift`. We should keep all the names consistent with our new naming convention (`Live Service`, as opposed to `Realtime` or `LiveStream`).